docs/content/dagster-plus/deployment/azure/blob-compute-logs.mdx

```diff
@@ -0,0 +1,119 @@
+ We'll do this with some azure CLI commands.
+
+First, we'll enable the cluster to use workload identity. This will allow the AKS agent to use a managed identity to access Azure resources.
+
+```bash
+az aks update --resource-group <resource-group> --name <cluster-name> --enable-workload-identity
+```
+
+Then, we'll create a new managed identity for the AKS agent, and a new service account in our AKS cluster.
+
+```bash
+az identity create --resource-group <resource-group> --name agent-id
+kubectl create serviceaccount dagster-agent-sa --namespace dagster-agent 
+```
+
+Now we need to federate the managed identity with the service account.
+
+```bash
+az identity federated-credential create \                                                                             
+  --name dagster-agent-federated-id \
+  --identity-name agent-id \
+  --resource-group <resource-group> \
+  --issuer $(az aks show -g <resource-group> -n <aks-cluster-name> --query "oidcIssuerProfile.issuerUrl" -otsv) \
+  --subject system:serviceaccount:dagster-agent:dagster-agent-sa
+```
+
+Finally, we'll edit our AKS agent deployment to use the new service account.
+
+```bash
+kubectl edit deployment <your-user-cloud-deployment> -n dagster-agent 
+```
+
+In the deployment manifest, add the following lines:
+
+```yaml
+metadata:
+  ...
+  labels:
+    ...
+    azure.workload.identity/use: "true"
+spec:
+  ...
+  template:
+    ...
+    spec:
+      ...
+      serviceAccountName: dagster-agent-sa
+```
+
+If everything is set up correctly, you should be able to run the following command and see an access token returned:
+```bash
+kubectl exec -n dagster-agent -it <pod-in-cluster> -- bash
+# in the pod
+curl -H "Metadata:true" "http://169.254.169.254/metadata/identity/oauth2/token?resource=https://storage.azure.com/"
+```
+
+## Step 2: Configure Dagster to use Azure Blob Storage
+
+Now, you need to update the helm values to use Azure Blob Storage for logs. You can do this by editing the `values.yaml` file for your user-cloud deployment.
+
+Pull down the current values for your deployment:
+
+```bash
+helm get values user-cloud > current-values.yaml
+```
+
+Then, edit the `current-values.yaml` file to include the following lines:
+
+```yaml
+computeLogs:
+  enabled: true
+  custom:
+    module: dagster_azure.blob.compute_log_manager
+    class: AzureBlobComputeLogManager
+    config:
+      storage_account: chriscomplogmngr
+      container: mycontainer
+      default_azure_credential:
+        exclude_environment_credential: false
+      prefix: dagster-logs-
+      local_dir: "/tmp/cool"
+      upload_interval: 30
+```
+    
+Finally, update your deployment with the new values:
+    
+```bash
+helm upgrade user-cloud dagster-cloud/dagster-cloud-agent -n dagster-agent -f current-values.yaml
+```
+
+## Step 3: Verify logs are being written to Azure Blob Storage
```

Comment:
I think it would be nice to include perhaps some troubleshooting steps as an addendum if the logs don't work. 

Ex:
How would the user see if:
- the credentials are wrong 
- the credentials do not have the necessary permissions
- the bucket does not exist or isn't writable?

Possibly we don't have a great story there, which would perhaps be follow up work we could add on later.

Reply:
yea the reality is I don't have a great story yet. I think we should do it as a follow up as we learn more.